server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -200,6 +236,11 @@ h1.usa-step-indicator__heading {
   max-width: 480px;
 }
 
+.cf-ul-basic {
```

Comment:
It's unclear to me that this should be in the theme styles, it seems very specific to the element it's styling below. Is there a way to use utility classes (.padding-left-205px) for this instead?